supervisor/v1/supervisor.proto

```diff
@@ -271,6 +274,15 @@ service Supervisor {
     // Dump the dataplane's current configuration
     rpc DumpConfig(DumpConfigRequest) returns (DumpConfigResponse) {}
 
+    // Create a snapshot
```

Comment:
This needs proper documentation. It's the CP's interface into the DP! You need to describe the semantics of the API, how and when to use it, is it idempotent, the return codes etc. See the documentation for the other functions above for inspiration.